app/tests/spec/lib/metrics.js

```diff
@@ -196,7 +198,7 @@ function (chai, $, p, Metrics, AuthErrors, Environment, sinon, _, WindowMock, Te
               assert.equal(windowMock.navigator.sendBeacon.getCall(0).args[0], '/metrics');
 
               var data = JSON.parse(windowMock.navigator.sendBeacon.getCall(0).args[1]);
-              assert.lengthOf(Object.keys(data), 22);
+              assert.lengthOf(Object.keys(data), 23);
```

Comment:
:+1: @vladikoff - this seems like a pretty ideal location to convert from the current style to a fixture like you often create for backend tests.